content/en/blog/_posts/2025-xx-xx-gang-scheduling.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: blog
+draft: true
+title: "Kubernetes v1.35: Introducing Workload Aware Scheduling"
+date: 2025-XX-XX
+slug: introducing-workload-aware-scheduling
+author: >
+  Maciej Skoczeń (Google),
+  Dominik Marciński (Google)
+---
+
+Scheduling large workloads is a much more complex and fragile operation than scheduling a single Pod,
+as it often requires considering all Pods together instead of scheduling each one independently.
+At the same time, the Pods that are part of such a workload are very often identical
+from the scheduling perspective, which fundamentally changes how this process should look.
+
+There are many custom schedulers adapted to perform workload scheduling efficiently,
+but considering how common and important workload scheduling is to Kubernetes users,
+it's high time to make workloads a first-class citizen for `kube-scheduler` and support them natively.
```

Comment:
Let's mention why we start it now, why "its high time"? (e.g., growth of ML workload usecase, many gang/WAS implementation in the community etc)